pkg/compiler/compiler.go

```diff
@@ -363,6 +364,9 @@ func CreateManifest(di *DebugInfo, o *Options) (*manifest.Manifest, error) {
 						name, len(ev.Parameters), len(argsList[i]))
 				}
 				for j := range ev.Parameters {
+					if ev.Parameters[j].Type == smartcontract.AnyType || ev.Parameters[j].Type == smartcontract.InteropInterfaceType {
```

Comment:
I'm not sure about `InteropInterfaceType`, when we clearly see some `Integer` emitted in place of `InteropInterfaceType` it's an error (unlike with `AnyType`, any is any).

Reply:
I had the same doubts, so fixed. But we still do not emit code for conversion to `InteropInterface` if it is declared in manifest, because it makes no sense.